test/e2e-tests.sh

```diff
@@ -97,6 +97,10 @@ kubectl replace cm "config-gc" -n ${SYSTEM_NAMESPACE} -f ${TMP_DIR}/config-gc.ya
 # Note that we use a very high -parallel because each ksvc is run as its own
 # sub-test. If this is not larger than the maximum scale tested then the test
 # simply cannot pass.
+if (( SHORT )); then
+  TEST_OPTIONS+=" -short"
```

Comment:
Is it a bit confusing for the -short option to actually only turn short mode on for the tests that run after this point in the file? I think I'd either add short to the options for all tests, or make the flag -short-scale-tests

Reply:
Agreed. Let's add the options for all tests.